examples/meter/meter.html

```diff
@@ -0,0 +1,169 @@
+      <div role="meter" tabindex="0" aria-valuenow="90" aria-valuemin="0" aria-valuemax="128"
+        aria-label="Disk Usage">
+        <div id="fill"></div>
+      </div>
+    </div>
+    <div role="separator" id="ex_end_sep" aria-labelledby="ex_end_sep ex_label" aria-label="End of"></div>
+  </section>
+  <section>
+      <h2 id="kbd_label">Keyboard Support</h2>
+
+      <p>It would not have any keyboard support since this is system generated information.</p>
+  </section>
+
+  <section>
+    <h2 id="rps_label">Role, Property, State, and Tabindex Attributes</h2>
+    <table aria-labelledby="rps_label" class="data attributes">
+      <thead>
+        <tr>
+          <th scope="col">Role</th>
+          <th scope="col">Attribute</th>
+          <th scope="col">Element</th>
+          <th scope="col">Usage</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr data-test-id="meter-role">
+          <th scope="row"><code>meter</code></th>
+          <td></td>
+          <td>
+            <code>div</code>
+          </td>
+          <td>
+            <ul>
+              <li>Identifies the element as a <code>meter</code> widget.</li>
+            </ul>
+          </td>
+        </tr>
+        <tr data-test-id="meter-tabindex">
+            <td></td>
+            <th scope="row">
+              <code>tabindex=&quot;0&quot;</code>
+            </th>
+            <td>
+              <code>div</code>
+            </td>
+            <td>
+              <ul>
+                <li>Includes the element in the tab sequence.</li>
+              </ul>
+            </td>
+          </tr>
+          <tr data-test-id="meter-aria-valuemin">
+              <td></td>
+              <th scope="row"><code>aria-valuemin="NUMBER"</code></th>
+              <td><code>div</code></td>
+              <td>
+                <ul>
+                  <li>Specifies a numeric value that is the minimum value the meter can have.</li>
+                </ul>
+              </td>
+            </tr>
+            <tr data-test-id="meter-aria-vlauemax">
+                <td></td>
+                <th scope="row"><code>aria-valuemax="NUMBER"</code></th>
+                <td><code>div</code></td>
+                <td>
+                  <ul>
+                    <li>Specifies a numeric value that is the maximum value the meter can have.</li>
+                  </ul>
+                </td>
+            </tr>
+            <tr data-test-id="meter-aria-valuenow">
+                <td></td>
+                <th scope="row"><code>aria-valuenow="NUMBER"</code></th>
+                <td><code>div</code></td>
+                <td>
+                  <ul>
+                    <li>A numeric value that is the current value of the meter</li>
+                  </ul>
+                </td>
+              </tr>
+              <tr data-test-id="meter-aria-label">
+                  <td></td>
+                  <th scope="row"><code>aria-label</code></th>
```

Comment:
Should probably have a visible label with `aria-labelledby` instead of using `aria-label`?